.binstar.yml

```diff
@@ -19,3 +19,11 @@ script:
   - conda build --quiet conda.recipe
 
 build_targets: conda
+
+---
+
+platform:
+  - win-32
```

Comment:
Did you open an issue for the win 32 failure... linux 32 issue should be solved in the next Anaconda Repo release... I think